packages/constants/src/lib/constants/mappers.ts

```diff
@@ -1,9 +1,25 @@
 import depd from 'depd';
 
-import { datilDev, datilTest, datil } from '@lit-protocol/contracts';
-
 import { LIT_NETWORK_VALUES } from './constants';
 
+// Legacy root import that drags every network/ABI blob into bundles; keep commented as a warning.
+// import { datil, datilDev, datilTest } from '@lit-protocol/contracts';
+
+// @ts-ignore -- TypeScript can't resolve the subpath because this package compiles to CJS, but runtime bundlers can.
+import { datil as datilContext } from '@lit-protocol/contracts/prod/datil.js';
```

Comment:
We should just be able to import this without the `.js` extension and have TS detect the types properly 🤔 
TS >4.9 uses the `exports` paths to find appropriate types 🤔 
You might need to remove the barrel file from the root `index.ts` entirely :+1:
We should also go ahead and remove the `typingVersions` export since our supported versions at this point is high enough that the `exports` key is what should be used anyway :+1: :)